app/_src/mesh/changelog.old.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Kong Mesh Changelog
-no_version: true
 ---
+<!-- TODO remove this file once https://github.com/Kong/docs.konghq.com/pull/5797 is merged as we can use the one directly from the Kong-Mesh repo -->
```

Comment:
Do we need to move this file?
We don't single-source changelogs, we just have one under `/<product>/changelog` and they use a different layout (no sidebar). I would keep it there for consistency. cc @lena-larionova

Reply:
+1 to this. Changelogs live outside of the `_src` directory because they're not versioned. Putting the file here would cause it to be versioned, and I don't think we want that.

Reply:
It's moved here just temporarily until we move to the auto generated one (https://github.com/Kong/docs.konghq.com/pull/5797) didn't want to do everything at once so did this.

Also I believe not having the sidebar is a little annoying (some goes in the changelog as part of looking at the docs (See https://kuma.io/docs/2.2.x ) for example but that's another discussion altogether.

Reply:
Can we leave the changelog changes out of this PR? We can take care of them in https://github.com/Kong/docs.konghq.com/pull/5797